Javascript/ants.js

```diff
@@ -132,37 +114,11 @@ function updateAntDescription(i) {
 function buyAnts(i) {
     let sacrificeMult = antSacrificePointsToMultiplier(player.antSacrificePoints);
     let type = "ant"
-    let tier = ""
     if (i === 1) {
         type = "reincarnation";
     }
-
-    switch (i) {
-        case 1:
-            tier = "first";
-            break;
-        case 2:
-            tier = "second";
-            break;
-        case 3:
-            tier = "third";
-            break;
-        case 4:
-            tier = "fourth";
-            break;
-        case 5:
-            tier = "fifth";
-            break;
-        case 6:
-            tier = "sixth";
-            break;
-        case 7:
-            tier = "seventh";
-            break;
-        case 8:
-            tier = "eighth";
-            break;
-    }
+    let ordinals = [null, "first", "second", "third", "fourth", "fifth", "sixth", "seventh", "eighth"]
```

Comment:
arrays start at 0, let's not continue bad practices if possible

hopefully @Pseudonian will remove all instances of this one day

Reply:
I guess my reasoning behind having that be 1-based was to be consistent with the rest of the codebase, but I went ahead and removed the whole ordinals array as there already is a 0-based version of it in variables.js.